includes/widgets/tabs.php

```diff
@@ -285,6 +315,34 @@ protected function _register_controls() {
 			]
 		);
 
+		$this->add_control(
+			'title_align',
+			[
+				'label' => __( 'Alignment', 'elementor' ),
+				'type' => Controls_Manager::CHOOSE,
+				'options' => [
+					'left' => [
+						'title' => __( 'Left', 'elementor' ),
+						'icon' => 'eicon-text-align-left',
+					],
+					'center' => [
+						'title' => __( 'Center', 'elementor' ),
+						'icon' => 'eicon-text-align-center',
+					],
+					'right' => [
+						'title' => __( 'Right', 'elementor' ),
+						'icon' => 'eicon-text-align-right',
+					],
+				],
+				'selectors' => [
+					'{{WRAPPER}} .elementor-tab-title' => 'text-align: {{VALUE}};',
```

Comment:
It might be better to use a CSS variable as our new approach

Reply:
This widget still does not work in the new method, I prefer to not do it only in one control.